2_isolation/README.md

```diff
@@ -28,3 +35,11 @@ $ python pit.py
 1. [Heuristics Analysis](heuristic_analysis.md)
 
 2. [Research Review](research_review.md)
+
+---
+
+## Screenshot
+
+![alt tag](https://raw.githubusercontent.com/ltfschoen/nd889/master/2_isolation/screenshots/screenshot.png)
```

Comment:
I've moved the screenshot to the "main" README.md, and moved the screenshot directory and file into the existing img folder